libyul/optimiser/FunctionCallFinder.h

```diff
@@ -32,16 +32,11 @@ namespace solidity::yul
  *
  * Prerequisite: Disambiguator
  */
-class FunctionCallFinder: ASTModifier
+class FunctionCallFinder
 {
 public:
 	static std::vector<FunctionCall*> run(Block& _block, YulString _functionName);
-private:
-	FunctionCallFinder(YulString _functionName);
-	using ASTModifier::operator();
-	void operator()(FunctionCall& _functionCall) override;
-	YulString m_functionName;
-	std::vector<FunctionCall*> m_calls;
+	static std::vector<FunctionCall const*> run(Block const& _block, YulString _functionName);
 };
 
 }
```

Comment:
At this point we do we even need the class? IMO it would be more natural to have just two static functions called `findFunctionCalls()`.